### PR TITLE
[ID] Update The Team

### PR DESCRIPTION
**Pull request by febrimuhnas**
Update translations of [en.md](https://osu.ppy.sh/help/wiki/People/The_Team?locale=en) in Indonesian

### CODE REVIEW

**Comment by febrimuhnas** [comment]
@peppy

File: wiki/People/The_Team/id.md [hunk 1]
```diff
@@ -1,56 +1,84 @@
...
-| Nama | Deskripsi |
-| ---- | ----------- |
-| [peppy](https://osu.ppy.sh/users/2) | Pemimpin proyek |
+Para pengguna yang tercantum di bawah adalah **tim inti osu!** dan bertanggung jawab untuk membantu menjadikan permainan ini luar biasa bagi semua orang.
 
-## Aktif
+| Nama | Peran <!-- TODO: "Featured Artist outreach" isn't a role but I'm not sure how to write it as one... --> |
+| :-- | :-- |
+| ![][flag_AU] **[peppy](https://osu.ppy.sh/users/2)** | **Pemimpin proyek** |
+| ![][flag_GB] [-Mo-](https://osu.ppy.sh/users/2202163) | Penasihat adegan *modding* dan *mapping*|
```
Comment (MegaApplePi) [inline wiki/People/The_Team/id.md]: Is there a reason for why "modding" and "mapping" are italicized? I tried looking for the diff and could not find if this was how it was originally in ID or EN.

**Comment by febrimuhnas** [comment]
because both of them haven't similar term in Indonesian. also In
Indonesian, foreign words are italicized.

Pada tanggal Min, 19 Jan 2020 10.32, Eric X <notifications@github.com>
menulis:

**Comment by MegaApplePi** [comment]
alright, @fajar13k just told me that as well

@fajar13k will take a look at this pull request soon

File: wiki/People/The_Team/id.md (hunk 1 above)
Comment (fajar13k) [inline wiki/People/The_Team/id.md]:
```suggestion
| ![][flag_GB] [-Mo-](https://osu.ppy.sh/users/2202163) | Penasihat adegan *modding* dan *mapping* |
```
spacing

File: wiki/People/The_Team/id.md [hunk 2]
```diff
@@ -1,56 +1,84 @@
...
+| ![][flag_US] [Chaos](https://osu.ppy.sh/users/2628870) | Penasihat moderasi dan komunitas |
+| ![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335) | Pemelihara proyek dan komunitas, Penjangkau Artis Unggulan |
+| ![][flag_JP] [flyte](https://osu.ppy.sh/users/3103765) | Pemimpin desain, pengamat Creative Cloud pengguna |
+| ![][flag_GB] [JBHyperion](https://osu.ppy.sh/users/4879508) | Penasihat adegan *modding* dan *mapping*|
```
Comment (fajar13k) [inline wiki/People/The_Team/id.md]:
```suggestion
| ![][flag_GB] [JBHyperion](https://osu.ppy.sh/users/4879508) | Penasihat adegan *modding* dan *mapping* |
```

File: wiki/People/The_Team/id.md [hunk 3]
```diff
@@ -1,56 +1,84 @@
...
+| ![][flag_AR] [juankristal](https://osu.ppy.sh/users/443656) | Penyelenggara dan administrator Piala Dunia osu! |
+| ![][flag_DE] [Mao](https://osu.ppy.sh/users/2204515) | Penasihat adegan *modding* dan *mapping*|
```
Comment (fajar13k) [inline wiki/People/The_Team/id.md]:
```suggestion
| ![][flag_DE] [Mao](https://osu.ppy.sh/users/2204515) | Penasihat adegan *modding* dan *mapping* |
```

File: wiki/People/The_Team/id.md [hunk 4]
```diff
@@ -1,56 +1,84 @@
...
+| ![][flag_JP] [nanaya](https://osu.ppy.sh/users/2387883) | Pengembang osu!web |
+| ![][flag_PH] [Nathanael](https://osu.ppy.sh/users/2295078) | ART umum, Penasihat moderasi |
+| ![][flag_AU] [nekodex](https://osu.ppy.sh/users/102) | Pengembang osu!web, [*official* osu! Artis Unggulan](https://osu.ppy.sh/beatmaps/artists/1) |
```
Comment (fajar13k) [inline wiki/People/The_Team/id.md]:
```suggestion
| ![][flag_AU] [nekodex](https://osu.ppy.sh/users/102) | Pengembang osu!web, [*official* Artis Unggulan osu!](https://osu.ppy.sh/beatmaps/artists/1) |
```

File: wiki/People/The_Team/id.md [hunk 5]
```diff
@@ -1,56 +1,84 @@
...
+| ![][flag_JP] [notbakaneko](https://osu.ppy.sh/users/10751776) | Pengembang osu!web |
+| ![][flag_DE] [Okoratu](https://osu.ppy.sh/users/1623405) | Penasihat adegan *modding* dan *mapping*|
```
Comment (fajar13k) [inline wiki/People/The_Team/id.md]:
```suggestion
| ![][flag_DE] [Okoratu](https://osu.ppy.sh/users/1623405) | Penasihat adegan *modding* dan *mapping* |
```

File: wiki/People/The_Team/id.md [hunk 6]
```diff
@@ -1,56 +1,84 @@
...
+| ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) | Penasihat adegan *modding* dan *mapping*, Penyelenggara Serikat Pe-mapping (Mappers' Guild), Penjangkau Artis Unggulan |
```
Comment (fajar13k) [inline wiki/People/The_Team/id.md]:
```suggestion
| ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) | Penasihat adegan *modding* dan *mapping*, Penyelenggara Serikat Pembuat Map (Mappers' Guild), Penjangkau Artis Unggulan |
```
Seems more appropriate.

File: wiki/People/The_Team/id.md [hunk 7]
```diff
@@ -1,56 +1,84 @@
...
-| [LuigiHann](https://osu.ppy.sh/users/1079) ([LuigiHann@Deviantart](http://luigihann.deviantart.com/)) | Desain skin keren, Kontribusi ke standar skin dan ikon set |
-| [mm201](https://osu.ppy.sh/users/30655) | Tim pengembang osu!, mm sliders |
-| [nuudles](https://osu.ppy.sh/users/21312) | Pengembang osu! di iPhone |
-| Phil | Forum desain kerja |
-| [Sarumaru](https://osu.ppy.sh/users/9427)  ([Sarumaru@deviantART.com](http://sarumaru.deviantart.com/)) | Konsep pippi asli dan desain karya seni |
-| [Sinistro](https://osu.ppy.sh/users/5530) | Menit Pertemuan, Kontribusi pertanyaan-pertanyaan umum |
-| [woc2006](https://osu.ppy.sh/users/1105845) | Tim pengembang osu!, Pengembang mode osu!mania |
-
-## Anggota grup
+Berikut ini adalah grup pengguna yang terdiri dari anggota komunitas osu! yang membantu menjaga dan memelihara osu!. Sebagian besar kelompok pengguna ini dapat dikenali dari warna forum, warna obrolan dalam game, judul profil, dan/atau lencana profil.
```
Comment (fajar13k) [inline wiki/People/The_Team/id.md]:
```suggestion
Berikut ini adalah grup pengguna yang terdiri dari anggota komunitas osu! yang membantu menjaga dan memelihara osu!. Sebagian besar kelompok pengguna ini dapat dikenali dari warna nama pengguna mereka di forum, warna obrolan dalam game, gelar pada profil, dan/atau lencana profil.
```

File: wiki/People/The_Team/id.md [hunk 8]
```diff
@@ -1,56 +1,84 @@
...
+| [osu! Alumni](osu!_Alumni) | Orang-orang yang dikenal atas kontribusi mereka setelah pindah/pensiun |
+| [Project Loved Team](Project_Loved_Team) | Mengenali beatmap yang paling disukai komunitas |
+| [Support Team](Support_Team) | Dukungan Teknis/Bantuan Permintaan Fitur |
+
+## Anggota tim inti yang kini tidak aktif
+
+| Nama | Peran |
+| :-- | :-- |
+| ![][flag_US] [awp](https://osu.ppy.sh/users/2650) | ART umum dan penolong, artis chibi. [Barrel Roll Weekly](http://brw.twinkfish.com/) |
+| ![][flag_NO] [cYsmix](https://osu.ppy.sh/users/272870) | Perancang audio, [*official* osu! Artis Unggulan](https://osu.ppy.sh/beatmaps/artists/2) |
```
Comment (fajar13k) [inline wiki/People/The_Team/id.md]:
```suggestion
| ![][flag_NO] [cYsmix](https://osu.ppy.sh/users/272870) | Perancang audio, [*official* Artis Unggulan osu!](https://osu.ppy.sh/beatmaps/artists/2) |
```

File: wiki/People/The_Team/id.md [hunk 9]
```diff
@@ -1,56 +1,84 @@
...
+| ![][flag_NZ] [deadbeat](https://osu.ppy.sh/users/128370) | Penyelenggara dan administrator Piala Dunia osu! |
```
Comment (fajar13k) [inline wiki/People/The_Team/id.md]:
```suggestion
| ![][flag_NZ] [deadbeat](https://osu.ppy.sh/users/128370) | Penyelenggara dan administrator osu! World Cup |
```
Keep it as it is.

File: wiki/People/The_Team/id.md [hunk 10]
```diff
@@ -1,56 +1,84 @@
...
+| ![][flag_US] [Derekku](https://osu.ppy.sh/users/91341) | ART umum, Manajer komunitas |
+| ![][flag_NZ] [Echo](https://osu.ppy.sh/users/431) | Pengembang osu!, penyedia integrasi IRC untuk obrolan dalam permainan, pengelola situs. [Blog](http://blog.echo.sh/) |
+| ![][flag_US] [HappyStick](https://osu.ppy.sh/users/256802) | Tuan rumah osu! Coffee Hour, Streamer Piala Dunia osu!, penyelenggara turnamen |
```
Comment (fajar13k) [inline wiki/People/The_Team/id.md]:
```suggestion
| ![][flag_US] [HappyStick](https://osu.ppy.sh/users/256802) | Tuan rumah osu! Coffee Hour, Penyiar osu! World Cup, penyelenggara turnamen |
```
Same.

File: wiki/People/The_Team/id.md [hunk 11]
```diff
@@ -1,56 +1,84 @@
...
+| ![][flag_NL] [Intermezzo](https://osu.ppy.sh/users/136842) | Pengembang osu!, provider of osz2 and p2p backend |
```
Comment (fajar13k) [inline wiki/People/The_Team/id.md]:
```suggestion
| ![][flag_NL] [Intermezzo](https://osu.ppy.sh/users/136842) | Pengembang osu!, penyedia *backend* osz2 dan p2p |
```

File: wiki/People/The_Team/id.md [hunk 12]
```diff
@@ -1,56 +1,84 @@
...
+| ![][flag_US] Jim | Original site designer. [Brand New Games](http://www.bravegamer.com/) |
```
Comment (fajar13k) [inline wiki/People/The_Team/id.md]:
```suggestion
| ![][flag_US] Jim | Desainer pertama situs web osu!. [Brand New Games](http://www.bravegamer.com/) |
```

File: wiki/People/The_Team/id.md [hunk 13]
```diff
@@ -1,56 +1,84 @@
...
+| ![][flag_DE] [Loctav](https://osu.ppy.sh/users/71366) | Penyelenggara dan administrator Piala Dunia osu!, pemimpin QAT, Manajer komunitas |
```
Comment (fajar13k) [inline wiki/People/The_Team/id.md]:
```suggestion
| ![][flag_DE] [Loctav](https://osu.ppy.sh/users/71366) | Penyelenggara dan administrator osu! World Cup, pemimpin QAT, Manajer komunitas |
```

File: wiki/People/The_Team/id.md [hunk 14]
```diff
@@ -1,56 +1,84 @@
...
+| ![][flag_US] [LuigiHann](https://osu.ppy.sh/users/1079) | Perancang skin epik, kontributor skin dan set ikon asali. [DeviantArt](https://luigihann.deviantart.com/) |
```
Comment (fajar13k) [inline wiki/People/The_Team/id.md]:
```suggestion
| ![][flag_US] [LuigiHann](https://osu.ppy.sh/users/1079) | Desainer skin handal, kontributor skin dan set ikon asli. [DeviantArt](https://luigihann.deviantart.com/) |
```
I think using "Desainer" and "handal" seems more suitable.
Also minor typo.

File: wiki/People/The_Team/id.md [hunk 15]
```diff
@@ -1,56 +1,84 @@
...
+| ![][flag_US] [mm201](https://osu.ppy.sh/users/30655) | Pengembang osu!, pembuat luncuran mm |
+| ![][flag_US] [nuudles](https://osu.ppy.sh/users/21312) | Pengembang osu! iPhone |
+| ![][flag_US] [Nyquill](https://osu.ppy.sh/users/682935) | Pengembang osu!, Penyelenggara Serikat Pe-mapping (Mappers' Guild) |
```
Comment (fajar13k) [inline wiki/People/The_Team/id.md]:
```suggestion
| ![][flag_US] [Nyquill](https://osu.ppy.sh/users/682935) | Pengembang osu!, Penyelenggara Serikat Pembuat Map (Mappers' Guild) |
```

File: wiki/People/The_Team/id.md [hunk 16]
```diff
@@ -1,56 +1,84 @@
...
+| ![][flag_US] Phil | Perancang forum |
+| ![][flag_US] [RBRat3](https://osu.ppy.sh/users/307202) | osu!painter, pembuat konten keren nan warbyazahh |
+| ![][flag_US] [Sarumaru](https://osu.ppy.sh/users/9427) | Perancang karya seni dan konsep pippi orisinil. [DeviantArt](https://sarumaru.deviantart.com/) |
```
Comment (fajar13k) [inline wiki/People/The_Team/id.md]:
```suggestion
| ![][flag_US] [Sarumaru](https://osu.ppy.sh/users/9427) | Perancang karya seni dan konsep pippi orisinal. [DeviantArt](https://sarumaru.deviantart.com/) |
```
KBBI

File: wiki/People/The_Team/id.md [hunk 17]
```diff
@@ -1,56 +1,84 @@
...
+| ![][flag_GR] [Sinistro](https://osu.ppy.sh/users/5530) |Manajer dan pemimpin komunitas awal, kontributor *meeting minutes* dan FAQ |
```
Comment (fajar13k) [inline wiki/People/The_Team/id.md]:
```suggestion
| ![][flag_GR] [Sinistro](https://osu.ppy.sh/users/5530) | Manajer dan pemimpin komunitas di era awal, kontributor *meeting minutes* dan FAQ |
```

**Comment by febrimuhnas** [comment]
I pick this:
Pengembang osu!, Pengembang mode osu!mania

Pada tanggal Min, 19 Jan 2020 11.19, Fajar Nazmi Fadillah <
notifications@github.com> menulis:

**Comment by fajar13k** [comment]
And apply it then alongside the review i mentioned above. Thank you.

**Comment by febrimuhnas** [comment]
@fajar13k Applied. Thankyou.

**Pull request merged**